style/base.css

```diff
@@ -231,3 +231,29 @@ ul.jp-notebook-job-output-formats-options {
 .jp-notebook-job-navigation-tab.active {
   border-block-end: 2px solid var(--jp-brand-color1);
 }
+
+
+.jp-collapsible-panel {
+  inline-size: 100%;
+}
+
+.jp-collapsible-panel-header {
```

Comment:
If we can constraint the title to be text, let's size the vertical height of this using the `--jp-size` variables in #13

Reply:
I really like the adaptable styling system you built around sizes. I need #13 merged to start using it. I can implement css variables with fallback values without #13 merged but some things like AvailableSizes type I'd need are not available